gamefixes-steam/287450.py

```diff
@@ -25,5 +25,16 @@ def main():
     # Set sound to alsa
     util.protontricks('sound=alsa')
 
+    # Install DirectPlay for reliable multiplayer connections, set overrides
+    util.protontricks('directplay')
+    util.winedll_override('dplaysvr.exe', 'n')
+    util.winedll_override('dplayx', 'n')
+    util.winedll_override('dpmodemx', 'n')
+    util.winedll_override('dpnet', 'n')
+    util.winedll_override('dpnhpast', 'n')
+    util.winedll_override('dpnhupnp', 'n')
+    util.winedll_override('dpnsvr.exe', 'n')
+    util.winedll_override('dpwsockx', 'n')
+
```

Comment:
I believe lines 30-37 may be redundant here as specifying the `directplay` verb already applies the native overrides for those files.